task3/UI/index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Opinion</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+    <header>
+      <img class="logo" src="Logo.png">
+      <input class="input-field" placeholder="Find your topic">
+      <button class="button my-profile"><p class="button-text">Andrei</p></button>
+      <button class="button log-out"><p class="button-text">Log out</p></button>
+      <button class="button menu"><img src="menu.png" class="menu icon"></button>
+      <div class="filtration">
+        <label class="user-name-filter" for="userName">User name<input id="userName" type="text" placeholder="name" class="user-name-filter input"></label>
+        <label class="date-from-filter" for="dateFrom">Date from:<input id="dateFrom" type="text" placeholder="02 03 2020" class="date input"></label>
+        <label class="date-to-filter" for="dateUntil">Date until:<input id="dateUntil" type="text" placeholder="02 03 2020" class="date input"></label>
```

Comment:
For date use type date

Reply:
And better to name it date To

Reply:
You need button "Apply Search". It can be added in the future, but you should understand that you will apply filters after clicking by this button